src/overlays/actors/ovl_En_Fu/z_en_fu.c

```diff
@@ -27,33 +39,274 @@ const ActorInit En_Fu_InitVars = {
     (ActorFunc)EnFu_Update,
     (ActorFunc)EnFu_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Fu/EnFu_Init.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Fu/EnFu_Destroy.s")
+static ColliderCylinderInit sCylinderInit = {
+    { COLTYPE_UNK10, 0x00, 0x11, 0x39, 0x10, COLSHAPE_CYLINDER },
+    { 0x00, { 0x00000000, 0x00, 0x00 }, { 0xFFCFFFFF, 0x00, 0x00 }, 0x00, 0x01, 0x01 },
+    { 30, 40, 0, { 0, 0, 0 } },
+};
+
+static Vec3f sMtxSrc = {
+    700.0f,
+    700.0f,
+    0.0f,
+};
+
+static UNK_PTR sEyesSegments[] = {
+    0x06005F20,
```

Comment:
Shouldn't these be pointers?
```suggestion
    D_06005F20,
```